src/base/components/DeviceProfile.js

```diff
@@ -17,6 +152,102 @@ class DeviveProfileTab extends React.Component {
 
+          <div className="row">
+            <div className="control-left control-panel col-sm-6">
+              <h1><i className="material-icons">notifications_active</i>Alerts</h1>              
+              <ToolTipLite className="tooltip-b" content="Available to Alpha testers only"><button class={this.state.devicestatus=== "off" ?  "btn btn-danger btn-sm" : "btn btn-success btn-sm"} type="button" id="Mote" ref="Mote" value={this.state.devicestatus}  onClick={this.setConfig} data-loading-text="Loading ...">Turn {this.state.devicestatus} Zolertia</button></ToolTipLite>
+            </div>
+            <div className="control-right control-panel col-sm-6">
+            <h1><i className="material-icons">info</i>Radio Parameters</h1>
+                            
+
+            <div class=" row">
+            <ToolTipLite className="tooltip-b" content="Transmission power in dBm.">
+              <label for="colFormLabelSm" class="col-sm-8 ">TXLevel:</label>
+              
+              <select class="col-sm-4" onChange={this.setTxConfig} value={this.state.tx_level}>
+                <option value="7">7</option>
+                <option value="5">5</option>
+                <option value="3">3</option>
+                <option value="1">1</option>
+                <option value="0">0</option>
+                <option value="-1">-1</option>
+                <option value="-3">-3</option>
+                <option value="-5">-5</option>
+                <option value="-7">-7</option>
+                <option value="-9">-9</option>
+                <option value="-11">-11</option>
+                <option value="-13">-13</option>
+                <option value="-15">-15</option>
+                <option value="-24">-24</option> 
+              </select>
+              </ToolTipLite>
+            </div>
+
+            <div class="row">
+            <ToolTipLite className="tooltip-b" content="Channel used for radio communication. The channel depends on the communication standard used by the radio">
+              <label for="colFormLabelSm" class="col-sm-8 ">Channel:</label>
+              
+              <select class="col-sm-4" onChange={this.setChConfig} value={this.state.cha}>
+                <option value="11">11</option>
+                <option value="12">12</option>
+                <option value="13">13</option>
+                <option value="14">14</option>
+                <option value="15">15</option>
+                <option value="16">16</option>
+                <option value="17">17</option>
+                <option value="18">18</option>
+                <option value="19">19</option>
+                <option value="20">20</option>
+                <option value="21">21</option>
+                <option value="22">22</option>
+                <option value="23">23</option>
+                <option value="24">24</option> 
+                <option value="25">25</option> 
+                <option value="26">26</option> 
```

Comment:
Hardcoded

Reply:
Its just a template now, in future it will not be hardcoded...